components/script/dom/serviceworkerglobalscope.rs

```diff
@@ -353,6 +351,7 @@ impl ServiceWorkerGlobalScope {
                     .pipeline_id(Some(pipeline_id))
                     .referrer_policy(referrer_policy)
                     .insecure_requests_policy(scope.insecure_requests_policy())
+                    // .has_trustworthy_ancestor_origin()
```

Comment:
Let's remove this. If it's a missing feature, we can file an issue for it instead.